### PR TITLE
nrf70_bm_lib: improve power ceil docs

### PR DESCRIPTION
**Pull request by ioannisg**
Improve TX power ceiling docs,
detailing how the power ceiling
configurations are normally
extracted.

### CODE REVIEW

**Comment by ioannisg** [comment]
@krish2718 can you take a look?

File: nrf70_bm_lib/include/nrf70_tx_pwr_ceil_dk.h [hunk 1]
```diff
@@ -6,10 +6,13 @@
  */
 
 /** @file
- * @brief nRF70 DK TX power ceiling configuration.
+ * @brief nRF7002 DK TX power ceiling configuration.
```
Comment (krish2718) [inline nrf70_bm_lib/include/nrf70_tx_pwr_ceil_dk.h:9]: it was for both nRF7002/nRF7001 DKs

File: nrf70_bm_lib/include/nrf70_tx_pwr_ceil_ek.h [hunk 2]
```diff
@@ -6,10 +6,13 @@
  */
 
 /** @file
- * @brief nRF70 EK TX power ceiling configuration.
+ * @brief nRF7002 EK TX power ceiling configuration.
```
Comment (krish2718) [inline nrf70_bm_lib/include/nrf70_tx_pwr_ceil_ek.h:9]: applicable for all EKs

File: nrf70_bm_lib/include/nrf70_tx_pwr_ceil_dk.h (hunk 1 above)
Comment (ioannisg) [reply nrf70_bm_lib/include/nrf70_tx_pwr_ceil_dk.h:9]: There is only the 7002 DK - the others are emulators

File: nrf70_bm_lib/include/nrf70_tx_pwr_ceil_ek.h (hunk 2 above)
Comment (ioannisg) [reply nrf70_bm_lib/include/nrf70_tx_pwr_ceil_ek.h:9]: Same comment here. I think it is confusing if we list non-physical boards. Thoughts?

**Comment by ioannisg** [comment]
@krish2718 added clarifications - pls take a look

File: nrf70_bm_lib/include/nrf70_tx_pwr_ceil_ek.h (hunk 2 above)
Comment (krish2718) [reply nrf70_bm_lib/include/nrf70_tx_pwr_ceil_ek.h:9]: Sure

**Review by krish2718 — APPROVED** [review]

**Pull request merged**